Code/Include/ROS2/Manipulator/MotorizedJoint.h

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+#pragma once
+
+#include "VehicleDynamics/DriveModels/PidConfiguration.h"
+#include <AzCore/Component/Component.h>
+#include <AzCore/Component/TickBus.h>
+
+namespace ROS2
+{
+
+    class MotorizedJoint
```

Comment:
Please add documentation
What does this class do, how it is intended to be used, etc.